svg/elements/textpath.json

```diff
@@ -0,0 +1,390 @@
+{
+  "svg": {
+    "elements": {
+      "textPath": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Element/textPath",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "3",
+              "notes": [
+                "Until Safari 6, textPath was not re-render when the referenced path was changed dynamically (see <a href='https://bugs.webkit.org/show_bug.cgi?id=15799'>bug 15799</a>'"
+              ]
+            },
+            "safari_ios": {
+              "version_added": "3",
+              "notes": [
+                "Until Safari 6, textPath was not re-render when the referenced path was changed dynamically (see <a href='https://bugs.webkit.org/show_bug.cgi?id=15799'>bug 15799</a>'"
+              ]
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "href": {
```

Comment:
Not sure all browsers actually support `href` in addition to `xlink:href` yet, though at least the current versions of Firefox, Chrome, Opera, IE, and Edge do so.

Reply:
Safari does too so I think it safe to assume support for all browsers.